cluster/centos/util.sh

```diff
@@ -316,10 +333,54 @@ function provision-node() {
 #   KUBE_TEMP
 function ensure-setup-dir() {
   kube-ssh "${1}" "mkdir -p ${KUBE_TEMP}; \
+                   mkdir -p ${KUBE_TEMP}/etcd-cert; \
                    sudo mkdir -p /opt/kubernetes/bin; \
                    sudo mkdir -p /opt/kubernetes/cfg"
 }
 
+function ensure-etcd-cert() {
```

Comment:
Yes, it's a good suggestion. Download the `cfssl` binaries is duplicate at least https://github.com/kubernetes/kubernetes/blob/master/cluster/common.sh#L908.